### ISSUE TITLE
Move accessibility note into screen reader tag

### ISSUE THREAD

**Issue by antobinary** [bigbluebutton/bigbluebutton#16554]
Currently we display the somewhat confusing "accessible" word beside the link to download an annotated presentation

![Screenshot from 2023-01-24 16-36-49](https://user-images.githubusercontent.com/6312397/214426435-8edc691d-bdc7-455b-a6c7-0537fea4cbb4.png)

"may be not accessible" here seems to imply not having access to the resource in the link. However, the intention is to warn screen reader users that the content they will download is likely not well formatted for use with screen reader (i.e. "Accessibility")

I checked with @KDSBrowne and from first glance it would be acceptable to move this screen-reader intended string to an `aria-label` or something similar / tooltip.

**BBB version:**
BigBlueButton 2.6.0-beta.7
